docs/process_joined_data.js

```diff
@@ -41,6 +42,11 @@ function colorAndRate(
   feature.properties['color'] = [255, 0, 255, opacity];
 }
 
+const KEY_BLACKLIST = Object.freeze(new Set(['system:index', 'color']));
```

Comment:
Can you add comments about why we expect these two strings to be here and why we don't want to show them. Maybe constant-ize 'color' and use above?

Reply:
Reworked logic a bit, made more general.